bin/plugin/cli.js

```diff
@@ -80,6 +84,13 @@ withOptions( program.command( 'build-standalone-plugins' ), buildPluginsOptions
 	)
 	.action( catchException( buildPluginsHandler ) );
 
+withOptions( program.command( 'test-standalone-plugins' ), testPluginsOptions )
```

Comment:
@10upsimon Can you run `npm run lint-js` and fix the error.

Reply:
@mukeshpanchal27 I got no changes at all when running `npm run lint-js` and my eslint is also not showing errors. Can you please confirm this again? What is the error you are seeing?

Reply:
I can confirm that I also am not getting errors, so this may be a configuration problem.